aws/resource_aws_batch_compute_environment_test.go

```diff
@@ -447,7 +473,8 @@ resource "aws_iam_role_policy_attachment" "aws_ec2_spot_fleet_role" {
 ########## security group ##########
 
 resource "aws_security_group" "test_acc" {
-  name = "tf_acc_test_batch_sg_%d"
+  name   = "tf_acc_test_batch_sg_%d"
+  vpc_id = "${aws_vpc.test_acc.id}"
```

Comment:
This was added to help produce a valid CE when `desired_vcpu` is non-zero. If I made code changes to the resource such that it wouldn't send a value does desired vCPUs, the CE would get into an invalid state citing that its associated security group wasn't in the specified VPC.